docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -27,22 +27,22 @@ Additionally, this feature can be configured to require touch for every Teleport
   Supported (`tsh`, `tctl`, and Teleport Connect):
     
     - Standard Teleport API requests (`tsh ls`, `tctl create`, etc.)
-    - Server Access
-    - Database Access
+    - Teleport SSH Service 
```

Comment:
This doesn't seem right to me. This docs page is attempting to say that hardware key support works for accessing SSH servers or databases. Why are we mentioning the specific service in the Teleport process here?